Nimble_Snapshots/CurrentTestCaseTracker.swift

```diff
@@ -17,12 +17,8 @@ import XCTest
 
 extension XCTestCase {
     var sanitizedName: String? {
-        guard let fullName = self.name else {
-            return nil
-        }
-
         let characterSet = CharacterSet(charactersIn: "[]+-")
-        let name = fullName.components(separatedBy: characterSet).joined()
+        let name = self.name.components(separatedBy: characterSet).joined()
```

Comment:
Interesting, thanks @marcgovi for the comment. I've been a little busy lately, I'l l try to continue the migration as far I can and update it for XCode 9 beta 6